platform/darwin/src/MGLLogging_Private.h

```diff
@@ -0,0 +1,20 @@
+#import "MGLLogging.h"
+
+#define mgl_log_info(message, ...)  mgl_log_with_type(MGLLoggingLevelInfo, __PRETTY_FUNCTION__, __LINE__, message, ##__VA_ARGS__)
```

Comment:
We should change these macros to uppercase, to match convention.

Reply:
I think it's worth us discussing whether we should have some (if not all) these macros defined if we have a `MGL_LOGGING_ENABLED` (or similar) defined, otherwise defined to nothing.

Reply:
> I think it's worth us discussing whether we should have some (if not all) these macros defined if we have a MGL_LOGGING_ENABLED (or similar) defined, otherwise defined to nothing.

That makes sense, but the way is designed is that it will be always present. Since is checking for the logging level it won't be activated unless the developer sets a level different than `MGLLoggingLevelNone` kind of the intention of `MGL_LOGGING_ENABLED` with the difference that can be activated in already deployed apps to trace down problems and avoiding to make a build with this flag enabled.